feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.ts

```diff
@@ -43,7 +61,13 @@ export class ImportEntriesDialogComponent {
   context$: Observable<AddOrderEntriesContext> =
     this.launchDialogService.data$.pipe(pluck('orderEntriesContext'));
 
-  constructor(protected launchDialogService: LaunchDialogService) {}
+  constructor(
+    protected el: ElementRef,
```

Comment:
This addition to the constructor requires updating the migration schematics. It's simple to do but requires non trivial work to do, so feel free to ping me on Slack for help. As a tip look at file `projects/schematics/src/migrations/5_0/constructor-deprecations/constructor-deprecations.ts` and look at the array `CONSTRUCTOR_DEPRECATIONS_DATA`. It's where we add components with a deprecated constructor

Reply:
I think this is the last comment I need to address. I'll reach out after I look at` constructor-deprecations.ts` .

Reply:
@BenjaminEmiliani   I took a stab at updating the migration schematics.  Can you take a look?
thx

Reply:
Usually, new dependencies are added after the previous ones, I'm not sure if it's a requirement, but it's better to check or replace it.

Reply:
I fixed the parameter list so it's in the preferred order.